rest/models/snapshot.go

```diff
@@ -10,6 +10,9 @@ type GetAllTickersSnapshotParams struct {
 
 	// A comma separated list of tickers to get snapshots for.
 	Tickers *string `query:"tickers"`
+
+	// Include OTC securities in the response. Default is false (don't include OTC securities).
+	IncludeOTC *bool `query:"include_otc"`
```

Comment:
for new request params, we should also add builder methods: `func (p GetAllTickersSnapshotParams) WithIncludeOTC(q bool) *GetAllTickersSnapshotParams`